### PR TITLE
docs: update backup-github example

### PR DESCRIPTION
**Pull request by antongolub**
```
fatal: remote error: 
  The unauthenticated git protocol on port 9418 is no longer supported
```

### CODE REVIEW

File: examples/backup-github.mjs [hunk 1]
```diff
@@ -25,11 +25,14 @@ if (process.env[token]) {
...
+let res = await fetch(`https://api.github.com/users/${username}/repos?per_page=1000`, {headers})
 let data = await res.json()
-let urls = data.map(x => x.git_url)
+let urls = data.map(x => x.git_url.replace('git://github.com/', 'git@github.com:'))
 
 await $`mkdir -p backups`
 cd('./backups')
 
-await Promise.all(urls.map(url => $`git clone ${url}`))
+await urls.reduce(async (m, url, i) => {
```
Comment (antonmedv) [inline examples/backup-github.mjs:35]: Maybe chunk array and wait in batches?

Comment (antongolub) [reply examples/backup-github.mjs:35]: I hit the rate limit, so single thread is more reliable.

Comment (antonmedv) [reply examples/backup-github.mjs:35]: Well, it’s just for learning purposes.

Comment (antonmedv) [reply examples/backup-github.mjs:35]: Also if per command - for loop is more readable.

Comment (antongolub) [reply examples/backup-github.mjs:35]: git clone is i/o in most and doesn't really benefit from 100 threads

Comment (antongolub) [reply examples/backup-github.mjs:35]: git clone is i/o in most and doesn't really benefit from 100 threads

Comment (antongolub) [reply examples/backup-github.mjs]: `reduce → for of`, ok

Comment (antonmedv) [reply examples/backup-github.mjs]: Will be cool to find an example to show await Promise.all with [].map usage.

**Pull request merged**